lib/data_models/movie.dart

```diff
@@ -0,0 +1,61 @@
+import '../utils/constants.dart';
+
+class Movie {
+  final bool adult;
+  final String backdrop;
+  final List<int> genres;
+  final int id;
+  final String originalLanguage;
+  final String originalTitle;
+  final String overview;
+  final double popularity;
+  final String poster;
+  final String releaseDate;
+  final String title;
+  final bool video;
+  final num voteAverage;
+  final int voteCount;
+
+  Movie({
+    required this.adult,
+    required this.backdrop,
+    required this.genres,
+    required this.id,
+    required this.originalLanguage,
+    required this.originalTitle,
+    required this.overview,
+    required this.popularity,
+    required this.poster,
+    required this.releaseDate,
+    required this.title,
+    required this.video,
+    required this.voteAverage,
+    required this.voteCount,
+  });
+
+  factory Movie.fromJson(Map<String, dynamic> moviesJson) {
+    return Movie(
+      adult: moviesJson['adult'] as bool,
+      backdrop: moviesJson['backdrop_path'] as String,
+      genres: List.from(moviesJson['genre_ids']),
+      id: moviesJson['id'] as int,
+      originalLanguage: moviesJson['original_language'] as String,
+      originalTitle: moviesJson['original_title'] as String,
+      overview: moviesJson['overview'] as String,
+      popularity: moviesJson['popularity'] as double,
+      poster: moviesJson['poster_path'] as String,
+      releaseDate: moviesJson['release_date'] as String,
+      title: moviesJson['title'] as String,
+      video: moviesJson['video'] as bool,
+      voteAverage: moviesJson['vote_average'] as num,
+      voteCount: moviesJson['vote_count'] as int,
+    );
+  }
+
+  String get backdropUrl =>
+      Constants.baseUrl + backdrop;
+  String get posterUrl => Constants.baseUrl + poster;
```

Comment:
You can use string interpolator like 
'${Constants.baseUrl}$backdrop'

Reply:
Perfect, fixed!